content/md/franklin-insights/franklin-insights-subscriptions.md

```diff
@@ -16,16 +16,33 @@ When you subscribe a product to Franklin, Franklin **collects and validates info
 
 You have 2 ways to subscribe products to Franklin:
```

Comment:
I would remove the ":" if we are gonna remove the bullet points.

```suggestion
You have 2 ways to subscribe products to Franklin.
```

Reply:
fixed